pages/404.js

```diff
@@ -1,30 +1,22 @@
-import Link from 'next/link';
-import { get404 } from '../lib/api';
-import { H1, P } from '../style/typography';
-import { getString } from '../lib/richText';
-
-export default function FourOhFour(errorPageData) {
-  const {
-    error_page_title: errorPageTitle,
-    error_text: errorText,
-  } = errorPageData.errorPageData;
+import { get404, getNavigation } from '../lib/api';
+import FourOhFourPage from '../components/four-oh-four-page/FourOhFourPage';
 
+export default function FourOhFour({ navigationData, errorPageData }) {
   return (
-    <>
-      <H1>{getString(errorPageTitle)}</H1>
-      <P>{getString(errorText)}</P>
-      <Link href="/">
-        <P>Click here to redirect to home page!</P>
-      </Link>
-    </>
+    <FourOhFourPage
```

Comment:
Are we planning on adding the footer too?

Reply:
from my chat with jay I don't think so